content/en/getting_started/integrations.md

```diff
@@ -0,0 +1,140 @@
+
+**instance** - You define and map the instance of whatever you are monitoring in the `conf.yaml` file. For example, in the [`http_check` integration][8], you're defining the name associated with the instance of the HTTP endpoint you are monitoring up and downtime. You can monitor multiple instances in the same integration, and you do that by defining all of the instances in the `conf.yaml` file.
+
+**metrics** - The list of what is collected from your system by each integration. You can find the metrics for each integration in that integrations `metadata.csv` file. For more information about metrics, see the [Metrics][9] developer page. You can also set up [custom metrics][10], so if the integration doesn't offer a metric out of the box, you can usually add it.
+
+**metadata.csv** - The file that lists and stores the metrics collected by each integration.
+
+**custom check** - If you have a unique system that you want to monitor, or if you're going to expand the metrics already sent by an integration, you can build a [custom check][11] to define and send metrics to Datadog. However, if you want to monitor a generally available application, public service, or an open source project, and we don't have an integration that already exists for it, you should consider [building a new integration][1] instead of a custom check.
+
+**tagging** - [Tags][12] are a way to add customization to metrics so that you can filter and visualize them in the most useful way to you.
+
+**logging** - If the system you are monitoring has logs, you can customize the logs you are sending to Datadog and use our [Logging Management solution][13] to manage and analyze them.
+
+## Setting up an integration
+
+The Datadog Agent package includes most integrations, so you don't need to download each individually.
+
+### API and Application keys
+
+In order to download the the Datadog Agent package, you need an [API and Application key][14]. You can manage your accounts API and Application keys in the [API Settings page][15] of the UI.
+
+### Installation
+
+If you want to connect with a cloud service provider, navigate to that provider on the [Integrations page][16] for specific instructions on how to connect. For most other integrations, start by installing the [Datadog Agent][17]. Most integrations are supported on our containerized agents: [Docker][18], and [Kubernetes][19]. After you've downloaded the Agent, go to the [Integrations page][16] section to find specific configuration instructions for individual integrations.
+
+### Configuration
+
+Configurations are more specifc to [individual integrations][16]. Configure most integrations by editing the `conf.yaml` file, in the `conf.d/` folder at the root of your Agent's configuration directory. Most integration have a sample `conf.yaml` that lists all available configuration options.
+
+For example, this is a sample `conf.yaml` for the `http_check` integration:
```

Comment:
Http check might not be the best fit since it's not a real "integration" in a sense that it doesn't actually connect to a given technology. Maybe using something more common and easy to setup like `apache` might be better.

Reply:
The idea is to explain the required and optional parameter logic.

Reply:
We should also explain in this example the Log collection logic at the bottom of the file.

Reply:
We should also explain how to set-up multiple instances in the same check configuration file.. If the first instance is defined for the local host, we could define the second one on a remote endpoint (this killer feature is often under-estimated by customers.